src/style.css

```diff
@@ -0,0 +1,102 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+button {
+  outline: none;
+  border: none;
+}
+
+i {
+  opacity: 0.8;
+  color: gray;
+}
+
+.main-section {
+  background-color: #fff;
+  border-radius: 5px;
+  margin: 10% 10%;
+  box-shadow: 3px 3px 3px 2px gray;
+  padding: 2% 2%;
+}
+
+.title {
+  display: flex;
```

Comment:
In your CSS, I noticed, you used a lot of `display:flex;` property , maybe you can create a class for that instead of repeating it many times. Also, you can create variables in root so in the future, if you want to change the colors, font sizes or anything, you can change them in just 1 place which is your root place, for example : 
``` 
:root {
--color-gray: gray;
--font-s1: 1rem;
--font-s2: 2rem;
--font-s3: 2.5rem;
}
.main-section {
font-size: var(--font-s2);
}
```